src/index.tsx

```diff
@@ -141,6 +143,21 @@ const Content: VFC<{ serverAPI: ServerAPI }> = ({}) => {
           }}
         />
       </PanelSectionRow>
+      <PanelSectionRow>
+        <ToggleField
+          checked={overlayPause}
+          label="Pause on overlay"
+          tooltip="Pause apps when interacting with Steam Overlay."
+          icon={<FaHome />}
+          onChange={async (state) => {
+            const settings = await backend.loadSettings();
+            settings.overlayPause = state;
+            await backend.saveSettings(settings);
+            setOverlayPause(state);
+            backend.resetStickyPauseStates();
```

Comment:
Does it make sense to reset the sticky states when enabling/disabling this feature ? The sticky states are enabled when "Pause on focus loss" is on but it might be off and "Pause on overlay" on.

Reply:
I'm ok with not resetting sticky state.